### DISCUSSION TITLE
Running shiori on a server and adding links with a client

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [go-shiori/shiori#573] [Q&A] [answered]
Running Shiori on a subdomain on my VPS and loving it so far. I want to know if the workflow that I imagined for Shiori is possible with its present design. 

I want to be able to keep Shiori running on my VPS so that I can access it from my phone but also use the CLI on my PC to add links to the VPS instance. Is there a way to make this happen?

**Comment by fmartingr** [comment, marked as answer]
Hey @lokesh-krishna, this is currently not possible (unless you ssh to your server, obviously) and it's one of the many things that I have under the radar. I would love to use `shiori --server https://.... add <url>` or something like that, since I have a hosted shiori as well, but we need to change several things before that. One of the first steps is #497.

**Reply by lokesh-krishna** [reply]
Thanks a lot! That's how I’d dreamed of it working and would love to see it happen done day.
